### DISCUSSION TITLE
Support for Microsoft Outlook Actionable Messages

### DISCUSSION THREAD

**Discussion by 108signals** [excid3/noticed#460] [Ideas]
Hey Chris,

Hope you’re doing great! We’ve been using your project for our app, and it’s awesome. I had an idea for a new feature that I think would be super helpful: support for [Microsoft Outlook Actionable Messages](https://learn.microsoft.com/en-us/outlook/actionable-messages/).

A lot of us in enterprise environments rely on Outlook, and having actionable messages would be a game-changer. It would let users interact with emails directly within Outlook, making things like approvals and updates so much easier.

Let me know what you think.

Thanks

**Comment by excid3** [comment]
If someone wants to implement this, it can be an easy plugin to noticed. It's hard for us to maintain it unless we were using it regularly.

**Reply by 108signals** [reply]
Hi Chris - thanks for the quick response. Would you be open to sponsorship for development of this plugin?

**Reply by 108signals** [reply]
Hi Chris - wanted to see if you'd be open to a sponsorship to create this plugin. Thanks
